src/renderer/Components/CreatingPackage/CreatingOrder/BookmarkToolbar/index.jsx

```diff
@@ -1,49 +1,117 @@
 import axios from "axios";
+import { useState } from "react";
 
 import usePackageStore from "@renderer/store";
+import Modal from "../../../Modal";
 
 import addingIcon from "@images/addingIcon.svg";
 import downloadIcon from "@images/downloadIcon.svg";
 
 function BookmarkToolbar() {
-  const getOrder = usePackageStore((state) => state.getOrder);
-  const BookmarkTarget = getOrder();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [bookmarks, setBookMarks] = useState([]);
+  const {
+    clientStatus: { isLogin },
+    getOrder,
+    updateOrder,
+  } = usePackageStore();
 
-  const handleAddBookmark = () => {
-    // TODO: 유저아이디 받아오는 로직 추가 구현 필요
-    axios.post(
-      `${import.meta.env.VITE_SERVER_URL}/${"user-id"}/bookmark`,
+  const userId = localStorage.getItem("userID");
+  const notifyLoginRequired = () => {
+    alert("로그인 유저 전용 기능입니다.");
+  };
+
+  const handleAddBookmark = async () => {
+    if (!isLogin) {
+      notifyLoginRequired();
+      return;
+    }
+    const BookmarkTarget = getOrder();
+    const formattedBookmarkData = {
+      ...BookmarkTarget,
+      attachmentType: "",
+      attachmentUrl: "",
+    };
+
+    await axios.post(
+      `${import.meta.env.VITE_SERVER_URL}/${userId}/bookmark`,
+      formattedBookmarkData,
+      {
+        headers: {
+          "Content-Type": "application/json",
+        },
+      },
+    );
+  };
+
+  const handleGetBookmark = async () => {
+    if (!isLogin) {
+      notifyLoginRequired();
+      return;
+    }
+    const BookmarkTarget = getOrder();
+    const result = await axios.post(
+      `${import.meta.env.VITE_SERVER_URL}/${userId}/bookmark`,
       BookmarkTarget,
       {
         headers: {
           "Content-Type": "application/json",
         },
       },
     );
+
+    setBookMarks(result);
```

Comment:
지금은 목업 데이터를 사용하고 있어서 상관없을텐데 실제 axios 를 사용해서 데이터를 받아오면 data 의 내용이 담기기 때문에 아래와 같이 바꾸어주면 될 것 같습니다. 👍
```suggestion
    setBookMarks(result.data);
```

Reply:
오! 감사합니다~!